src/m_menu.c

```diff
@@ -3194,8 +3200,9 @@ setup_menu_t stat_settings1[] =  // Status Bar and HUD Settings screen
 
   {"Fullscreen HUD", S_SKIP|S_TITLE, m_null, M_X, M_SPC},
 
-  {"HUD Type", S_CHOICE, m_null, M_X, M_SPC, {"hud_type"}, 0, NULL, str_hudtype},
+  {"HUD Type", S_CHOICE, m_null, M_X, M_SPC, {"hud_type"}, 0, M_UpdateHUDModeItem, str_hudtype},
   {"HUD Mode", S_CHOICE, m_null, M_X, M_SPC, {"hud_active"}, 0, NULL, str_hudmode},
+  {"Widescreen", S_YESNO, m_null, M_X, M_SPC, {"hud_widescreen"}, 0, HU_Start},
```

Comment:
Screenshots: [HUD](https://github.com/fabiangreffrath/woof/assets/56656010/6ee90d20-6e9c-4c6a-a858-00e42d0547e8), [Widgets](https://github.com/fabiangreffrath/woof/assets/56656010/15086712-696e-4d55-bd7c-dfc7104dfda5)

Reply:
> Screenshots: [HUD][Widgets]

Are all the combinations I presented here still possible?

https://github.com/fabiangreffrath/woof/wiki/HUD-modes

Reply:
Every combination is still possible except these two:

[hud_active = 1, hud_widescreen_widgets = 1, crispy_hud = 1](https://private-user-images.githubusercontent.com/6449237/301779009-8dd18eef-112c-43a6-a72f-a4ccf5338a9b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzkwNjUsIm5iZiI6MTcwNjg3ODc2NSwicGF0aCI6Ii82NDQ5MjM3LzMwMTc3OTAwOS04ZGQxOGVlZi0xMTJjLTQzYTYtYTcyZi1hNGNjZjUzMzhhOWIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDJUMTI1OTI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OTI3NWU1OTEyMWY4NDZmODU3NmM4ZDBiOWU5N2JlMmY2MTkwODdlNGM0YWFlZGY0NTk4Y2VkMzUwZTU0ZTY2NyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.bViD-DuLnKOD5CbMExzF0Vn6vCdIMLD-pf3v1G-YKjg)

[hud_active = 2, hud_widescreen_widgets = 0, crispy_hud = 1](https://private-user-images.githubusercontent.com/6449237/301778663-a6e5f3d3-bf64-4600-90b5-1826e3852b59.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDY4NzkwNjUsIm5iZiI6MTcwNjg3ODc2NSwicGF0aCI6Ii82NDQ5MjM3LzMwMTc3ODY2My1hNmU1ZjNkMy1iZjY0LTQ2MDAtOTBiNS0xODI2ZTM4NTJiNTkucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDJUMTI1OTI1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MjljMmZjMzQ1ZTRkZDIxNGM1Nzk4MDBiMGRkYTU4NThmMjE3NTMwOTI0NzBlM2ZhY2JlMTAyN2E5MDc4YmIwNiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.80UjkXFV54e2YOg_erm3egJn3m-unAh5uAcrmUhJeec)

I think it's okay, they don't make any sense to me.

Reply:
So, no "mixed mode". Yes, I think that's okay.